### DISCUSSION TITLE
"authenticatedUser: null" in server logs even when Im authenticated

### DISCUSSION THREAD

**Discussion by tananga** [serverpod/serverpod#3090] [General]
Hello. I have few endpoints:

Future<VenueList> list(Session session, {int limit = 10, int page = 1}) async {

and 

Future<Venue?> save(Session session, Venue venue) async {

save requires user to be authenticated. list dont care if user is authenticated or not.
When Im running my local server with dart bin/main.dart --apply-migrations I see this logs
CALL: venue.save duration: 9ms numQueries: 2 authenticatedUser: 2
CALL: venue.list duration: 10ms numQueries: 2 authenticatedUser: null

I think that It would be nice to see in logs that user with id 453 executed endpoint even if this endpoint dont require authentication

**Comment by SandPod** [comment]
This is being discussed in the issue here: https://github.com/serverpod/serverpod/issues/3091

**Reply by tananga** [reply]
Closed
